src/App.js

```diff
@@ -11,6 +13,8 @@ function App() {
           <Routes>
             <Route path="/" element={<Home />} />
             <Route path="/home" element={<Home />} />
+            <Route path="/sidbar" element={<SidBar />} />
+            <Route path="/PlannerCard" element={<PlannerCard />} />
```

Comment:
Quando for subir o src/App.js, matém ele limpo, apenas com o import da home no máximo, assim evita conflitos de merge